pkg/crcontrollers/csibmnode/controller.go

```diff
@@ -329,6 +368,23 @@ func (bmc *Controller) reconcileForCSIBMNode(bmNode *nodecrd.CSIBMNode) (ctrl.Re
 		}
 	}
 
+	if !bmNode.GetDeletionTimestamp().IsZero() {
+		bmc.disableForNode(k8sNode.Name)
```

Comment:
What will happen if Operator restarts during CRs removal?

Reply:
When operator starts he gets `Create Event` for each node. When operator handles these events he sets `bmc.enableForNode(k8sNode.Name)` for each node. Then operator gets reconcile (update) for CSIBMNode CRs and proceed with removing as it was before restart.

Reply:
Ok, got it. Thanks!